README.md

```diff
@@ -462,35 +462,23 @@ The entire implementation is purposefully under the `internal` package to not ex
 
 *Question:* How do I download `protoc` ahead of time as part of a Docker build/CI pipeline?
 
-*Answer*: We used to have a command that did this, but removed it for simplicity and because the command as implemented did not properly
-read the configuration file to figure out what version of `protoc` to download. We may re-add this command in the future, however
-here is a technique to accomplish this, including as a `RUN` directive for Docker:
+*Answer*: `prototool cache update`.
 
-```bash
-# the first rm -rf call is not needed if this is a RUN directive for Docker
-rm -rf /tmp/prototool-bootstrap && \
-  echo $'protoc\n  version: 3.6.1' > /tmp/prototool-bootstrap/prototool.yaml && \
-  echo 'syntax = "proto3";' > /tmp/prototool-bootstrap/tmp.proto && \
-  prototool compile /tmp/prototool-bootstrap && \
-  rm -rf /tmp/prototool-bootstrap
-```
-
-The better way to do this as part of a bash script would be to use `mktemp -d` i.e.:
+You can pass both `--cache-path` and `--config-data` flags to this command to customize the invocation.
 
 ```bash
-#!/bin/bash
-
-set -euo pipefail
-
-TMPDIR="$(mktemp -d)"
-trap 'rm -rf "${TMPDIR}"' EXIT
-
-echo $'protoc\n  version: 3.6.1' > "${TMPDIR}/prototool.yaml"
-echo 'syntax = "proto3";' > "${TMPDIR}/tmp.proto"
-prototool compile "${TMPDIR}"
+# Basic invocation which will cache using the default behavior. See prototool help cache update for more details.
+prototool cache update
+# Cache to a specific directory path/to/cache
+prototool cache update --cache-path path/to/cache
+# Cache using custom configuration data instead of finding a prototool.yaml file using the file discovery mechanism
+prototool cache update --config-data '{"protoc":{"version":"3.6.1"}}'
 ```
 
-But for Darwin or Linux, the above should work.
+There is also a command `prototool cache delete` which will delete all cached assets of `prototool`,
```

Comment:
I feel as if this behavior could lead to some frustrating behavior, unless they read this exact line of the README.

If you run:
```
prototool cache update --cache-path /tmp
prototool cache delete
prototool cache update --cache-path /tmp
```
You would expect the third command to download `protoc` again, however that won't be the case, the cache will be completely populated.

I understand the reasoning for keeping the option available, so perhaps the solution here is to add to the description of `--cache-path` that the user is expected to manage this folder, and to `cache delete` that it will only clean up the default cache.

Reply:
I could see the confusion if users expected prototool to maintain some kind of config that stored flag invocations, but it doesn't and I wouldn't expect the second command to do anything to my custom cache path since I don't specify the path on `delete`. However, I added a line to the documentation about this.